.travis.yml

```diff
@@ -1,11 +1,18 @@
 language: go
+sudo: required
 
 go:
   - "1.10"
+env:
+  global:
+  - PATH=$GOROOT/bin:$GOPATH/bin:$PATH
 
 before_install:
+   # Install Glide
+  - mkdir -p "${GOPATH}/bin"
+  - curl https://glide.sh/get | bash
```

Comment:
why ``glide``  is installed? do we need to update vendor folder during testing?

Reply:
I've seen some issues on travis that make triggered glide update(as glide files were changed) and failed. I would prefer not to run glide in travis at all and have the vendor up to date.